packages/contracts/contracts/L2/predeploys/TssRewardContract.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "./iTssRewardContract.sol";
+
+/* Library Imports */
+import { SafeMath } from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+/* Interface Imports */
+
+/* External Imports */
+
+/**
+ * @title TssRewardContract
+ * @dev Collect L2 block gas reward per block and release to batch roll up tss members.
+ */
+contract TssRewardContract is ITssRewardContract {
+    using SafeMath for uint256;
+
+    mapping(uint256 => uint256) public ledger;
+    address public deadAddress;
+    address payable public owner;
```

Comment:
We can extend `@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol` to check the owner.